features/305-microservice/microservice.md

```diff
@@ -83,3 +116,9 @@ In the future, it is possible to have configurable replicas or even auto horizon
 ## Routing to the microservice
 
 The existing routing mechanism for cloud code is enhanced to support routing to microservices.
+
+## Preconfigured environment
+
+Every `runtime` and `runtime_version` has its own template. The template should contain a `Dockerfile`. If the template requires scripts or other files, we should give special name to folders or files, to avoid conflict with the user source code files.
```

Comment:
I think we should reserve and specify a directory (e.g. `.skygear`) to store the scripts/dependencies in template.